### DISCUSSION TITLE
Invalid time value

### DISCUSSION THREAD

**Discussion by gokmenozkn** [vercel/next.js#17469] [Help] [answered]
![next-error](https://user-images.githubusercontent.com/56534538/94677836-3e12eb80-0326-11eb-986d-0152e7ecee4a.png)
I'm getting this error. How can I fix this?

**Comment by jamesmosier** [comment, marked as answer]
Hi @gokaygurcan. Assuming you are using `date-fns`, that error occurs when your `date` is not a valid date. If you `console.log(date)` you'll likely see that it is invalid. That usually means the `dateString` is not valid to be used inside `parseISO`

**Reply by gokaygurcan** [reply]
Close enough, but wrong handle :) Should be @gokmenozkn I assume.

**Reply by jamesmosier** [reply]
Oops! Sorry about that! 😄

**Reply by gokmenozkn** [reply]
I used sanity example from your repo. I didn't make any changes. I followed every step. But this issue occured. @jamesmosier I can't log the date by the way.

**Reply by jamesmosier** [reply]
Why can't you log the date?

```js
import { parseISO, format } from 'date-fns'

export default function Date({ dateString }) {  
  const date = parseISO(dateString)
  console.log('date', date)
  console.log('dateString', dateString)
  return <time dateTime={dateString}>{format(date, 'LLLL  d, yyyy')}</time>
}
```

then check your terminal and browser logs

**Reply by gokmenozkn** [reply]
It says `undefined`. But I chose the date on sanity.

**Reply by jamesmosier** [reply]
If  you log out `post.date` in [pages/posts/[slug].js](https://github.com/vercel/next.js/blob/canary/examples/cms-sanity/pages/posts/%5Bslug%5D.js#L19) is that undefined as well? 

If so then the next place you'd want to look into is [lib/api.js](https://github.com/vercel/next.js/blob/816798569a56c97108ecff37a85e6a3fd85648ab/examples/cms-sanity/lib/api.js#L51-L72) to see why the `date` field is not coming back in the query.

**Reply by agrittiwari** [reply]
Thanks, @jamesmosier . It helped me out.

**Comment by Kevin-Roberts** [comment]
I ran into this issue and the problem was that I had a .md.swp file in my posts directory because the file was open in vim. Removing the swap file fixed it.

**Comment by acomathes** [comment]
Another possible reason for getting this error is having the Dark Reader browser plugin turned on. In my case, turning it off and doing a page reload did the trick.

**Reply by itsbrex** [reply]
Odd that this would be cause that issue. 

What I'm seeing is dates being rendered as DD-1 (i.e. instead of April 1st it shows March 31st)? Could this be related?
